app/assets/stylesheets/moderators.scss

```
@@ -155,8 +159,446 @@
   padding: 30px 10px;
 }
 
+// these styles are to correct height and margin issues in the iframe
+.moderations-actions_panel {
+  min-height: initial;
+  height: 100%;
+  #page-content-inner {
+    margin-bottom: 0;
+    height: 100%;
+  }
+}
+
 .mod-container {
-  padding: 30px 0px;
+  display: flex;
+  flex-direction: column;
+  background-color: var(--base-0);
+  border: none;
+  box-shadow: none;
+  margin-bottom: 0;
+  height: 100%;
+  .circle {
+    border-radius: 100%;
+    &.centered-icon {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+    }
+  }
+
+  header {
+    &.top-header {
+      background-color: var(--base-inverted);
+      padding: var(--su-4) var(--su-4) 0;
+      display: flex;
+      flex-direction: column;
+    }
+    &.other-things {
+      padding: var(--su-4);
+      h1 {
+        text-transform: uppercase;
+        font-size: var(--fs-s);
+      }
+    }
+    h1 {
+      font-size: var(--fs-l);
+      font-weight: var(--fw-bold);
+    }
+    h2 {
+      font-size: var(--fs-base);
+      font-weight: var(--fw-normal);
+      color: var(--base-70);
+    }
+    button.close-actions-panel {
+      border: none;
+      width: 40px;
+      height: 40px;
+      position: absolute;
+      right: var(--su-4);
+      top: var(--su-4);
+      background-color: var(--base-10);
+    }
+  }
+  .reactions-container {
+    padding: var(--su-4);
+    background-color: var(--base-inverted);
+  }
+
+  .thumb-reactions-container {
+    display: grid;
+    grid-gap: var(--su-4);
+    grid-template-columns: 1fr 1fr;
+  }
+
+  .reaction-button {
+    position: relative;
+    background: white;
+    border-color: white;
+    border: 0;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    height: var(--su-10);
+    padding: var(--su-4) var(--su-6);
+    border-radius: var(--radius);
+
+    &:hover {
+      background: rgba(59, 73, 223, 0.05);
+    }
+
+    &:hover .reaction-button-circle {
+      border-color: var(--accent-brand);
+    }
+  }
+
+  .reaction-button-circle {
+    position: relative;
+    display: flex;
+    align-items: center;
+    justify-content: center;
```

See https://developer.mozilla.org/en-US/docs/Web/CSS/place-items

```suggestion
    place-items: center;
```